functional/FunctionalTest.java

```diff
@@ -1692,6 +1697,13 @@ public static void main(String[] args) {
       System.exit(-1);
     }
 
+    if (System.getenv("MINIO_MINT") != null) {
```

Comment:
There is no such variable `MINIO_MINT` in Mint env . I think we can use `DATA_DIR` here.